src/pages/saksbehandling/sakintro/Sakintro.tsx

```diff
@@ -340,4 +345,47 @@ const LukkedeOgAvslåtteSøknader = (props: { lukkedeOgAvslåtteSøknader: Søkn
     );
 };
 
+const AvslåtteSøknader = (props: { avslåtteSøknader: Søknad[]; intl: IntlShape }) => {
```

Comment:
Denna ser ut att vara exakt likt lukkedesøknader, bara att den ena property:n har ett annerledes navn?

Reply:
Skal være helt likt ut ja. Dette er en egen bolk for avslåtte søknader